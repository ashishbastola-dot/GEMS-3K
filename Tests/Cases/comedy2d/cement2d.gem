GeoSys-MMP  Material Medium Properties ------------------------------------
#GEM_PROPERTIES
  $GEM_THREADS
   8
  $GEM_INIT_FILE
    I1SetUp-dat.lst    ; only file name, no path
  $FLAG_NODE_ELEMENT
    0    ; 0-node based; 1-elem based;
  $FLAG_POROSITY_CHANGE
    1    ; 0-not coupled;1=coupled;
  $MIN_POROSITY
    1.e-6
  $MAX_POROSITY
    1.0
  $FLAG_COUPLING_HYDROLOGY
    1    ; 0-not coupled;1=coupled;
  $TEMPERATURE_GEM
    298.15       ; temperature for GEMS in degree Celsius
  $GEM_SMART
     0 ; do not set to one if kinetics is used
  $TRANSPORT_B
     1 ; 1: do transport of b vector
  $MAX_FAILED_NODES
     50 ; number of nodes alowed to fail 
  $MY_SMART_GEMS
    -1.0e-30
 
  $KINETIC_GEM     ; wie Traber and Maeder
   Portlandite 1   1     0.0 0.0 0.0   0.0  -5.0  0.0  0.0 1.0  1.0 1.0  0.0 1.0   H+  0.0  0.0 0.0
   3 1000.0
 $KINETIC_GEM     ; wie Traber and Maeder
   CaOxaHyd 1   1     0.0 0.0 0.0   0.0  -5.0  0.0  0.0 1.0  1.0 1.0  0.0 1.0   H+  0.0  0.0 0.0
   3 1000.0

#STOP




