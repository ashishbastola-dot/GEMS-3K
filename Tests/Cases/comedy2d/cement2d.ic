GeoSys-IC  Initial Conditions ------------------------------------------------

#STOP

#INITIAL_CONDITION
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 10.0

#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE
  1-C 
 $GEO_TYPE
  DOMAIN  
 $DIS_TYPE
  CONSTANT  2.99840439476672e-04
 
#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE
  2-Ca 
 $GEO_TYPE
  DOMAIN  
 $DIS_TYPE
  CONSTANT  5.99680878953344e-04
 
#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE
  3-Cl 
 $GEO_TYPE
  DOMAIN  
 $DIS_TYPE
  CONSTANT  7.19617054744013e-04
 
#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE
  4-H 
 $GEO_TYPE
  DOMAIN  
 $DIS_TYPE
  CONSTANT  3.32873105268756e+04
 
#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE
  5-Na 
 $GEO_TYPE
  DOMAIN  
 $DIS_TYPE
  CONSTANT  2.99840439476672e-04
 
#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE
  6-O 
 $GEO_TYPE
  DOMAIN  
 $DIS_TYPE
  CONSTANT  7.82864434353260e+04
 
#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE
  7-Oxa 
 $GEO_TYPE
  DOMAIN  
 $DIS_TYPE
  CONSTANT  2.99840439476672e-04
 
#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE
  8-Si 
 $GEO_TYPE
  DOMAIN  
 $DIS_TYPE
  CONSTANT  3.08213907427230e+04
 
#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE
  9-Zz 
 $GEO_TYPE
  DOMAIN  
 $DIS_TYPE
  CONSTANT  0.00000000000000e+00
 

#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE
  1-C 
 $GEO_TYPE
  SURFACE Q2 
 $DIS_TYPE
  CONSTANT  1.89715158511924e-04
 
#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE
  2-Ca 
 $GEO_TYPE
  SURFACE Q2
 $DIS_TYPE
  CONSTANT  3.31775667455921e+03
 
#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE
  3-Cl 
 $GEO_TYPE
  SURFACE Q2 
 $DIS_TYPE
  CONSTANT  1.89715158511924e-04
 
#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE
  4-H 
 $GEO_TYPE
  SURFACE Q2
 $DIS_TYPE
  CONSTANT  2.76970722798913e+04
 
#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE
  5-Na 
 $GEO_TYPE
  SURFACE Q2 
 $DIS_TYPE
  CONSTANT  1.89715158511924e-04
 
#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE
  6-O 
 $GEO_TYPE
  SURFACE Q2 
 $DIS_TYPE
  CONSTANT  7.88950893033171e+04
 
#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE
  7-Oxa 
 $GEO_TYPE
  SURFACE Q2
 $DIS_TYPE
  CONSTANT  1.89715158511924e-04
 
#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE
  8-Si 
 $GEO_TYPE
  SURFACE Q2 
 $DIS_TYPE
  CONSTANT  3.08643962523970e+04
 
#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE
  9-Zz 
 $GEO_TYPE
  SURFACE Q2  
 $DIS_TYPE
  CONSTANT  0.00000000000000e+00
 
#STOP

