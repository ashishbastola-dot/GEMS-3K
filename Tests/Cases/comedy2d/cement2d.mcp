#COMPONENT_PROPERTIES ; comp1 
 $NAME  
  1-C 
 $MOBILE
  1;  MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  3.333333e-9
 
#COMPONENT_PROPERTIES ; comp2 
 $NAME  
  2-Ca 
 $MOBILE
  1;  MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  3.333333e-9
 
#COMPONENT_PROPERTIES ; comp3 
 $NAME  
  3-Cl 
 $MOBILE
  1;  MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  3.333333e-9
 
#COMPONENT_PROPERTIES ; comp4 
 $NAME  
  4-H 
 $MOBILE
  1;  MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  3.333333e-9
 
#COMPONENT_PROPERTIES ; comp5 
 $NAME  
  5-Na 
 $MOBILE
  1;  MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  3.333333e-9
 
#COMPONENT_PROPERTIES ; comp6 
 $NAME  
  6-O 
 $MOBILE
  1;  MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  3.333333e-9
 
#COMPONENT_PROPERTIES ; comp7 
 $NAME  
  7-Oxa 
 $MOBILE
  1;  MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  3.333333e-9
 
#COMPONENT_PROPERTIES ; comp8 
 $NAME  
  8-Si 
 $MOBILE
  1;  MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  3.333333e-9
 
#COMPONENT_PROPERTIES ; comp9 
 $NAME  
  9-Zz 
 $MOBILE
  0;  MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 
#STOP



#COMPONENT_PROPERTIES  ; comp1
 $NAME
  Cl- ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  3.333333333e-9             	; diffusion model type, diffusion constant
#STOP
