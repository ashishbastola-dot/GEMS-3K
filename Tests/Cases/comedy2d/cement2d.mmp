GeoSys-MMP  Material Medium Properties ---; Please keep same number of records in *.msp file.
#MEDIUM_PROPERTIES
 $GEOMETRY_DIMENSION
  2
 $GEOMETRY_AREA
  1.000000e+000
 $POROSITY
  15  0.30
 $TORTUOSITY
  1   1.000000e+000
 $PERMEABILITY_TENSOR                  
   ISOTROPIC 1.0e-5  ; ISOTROPIC  1.0194e-12  ; unit: m2
 $MASS_DISPERSION
  1  0.02  0.02
 $DENSITY
  1  1800.0
#MEDIUM_PROPERTIES
 $GEOMETRY_DIMENSION
  2
 $GEOMETRY_AREA
  1.000000e+000
 $POROSITY
  15  0.19
 $TORTUOSITY
  1   1.578949e+000
 $PERMEABILITY_TENSOR
  ISOTROPIC 1.64e-6   ;  ISOTROPIC  1.6718e-13  ; unit: m2
 $MASS_DISPERSION
  1  0.02  0.02
 $DENSITY
  1  1800.0
#MEDIUM_PROPERTIES
 $GEOMETRY_DIMENSION
  2
 $GEOMETRY_AREA
  1.000000e+000
 $POROSITY
  15  0.30
 $TORTUOSITY
  1   1.000000e+000
 $PERMEABILITY_TENSOR
   ISOTROPIC 1.0e-5   ;  ISOTROPIC   1.0194e-12  ; unit: m2
 $MASS_DISPERSION
  1  0.02  0.02
 $DENSITY
  1  1800.0
 #STOP
