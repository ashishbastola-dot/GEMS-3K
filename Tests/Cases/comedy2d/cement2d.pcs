GeoSys-PCS: Processes ------------------------------------------------
#PROCESS
 $PCS_TYPE
 GROUNDWATER_FLOW
 $NUM_TYPE
   NEW   
 $RELOAD
  3 250  ; write restart file every 100th timestep
#PROCESS
 $PCS_TYPE
  MASS_TRANSPORT
 $NUM_TYPE
   NEW    
 $RELOAD
  3 250  ; write restart file every 100th timestep
#PROCESS
 $PCS_TYPE
  MASS_TRANSPORT
 $NUM_TYPE
   NEW    
 $RELOAD
  3 250  ; write restart file every 100th timestep
#PROCESS
 $PCS_TYPE
  MASS_TRANSPORT
 $NUM_TYPE
   NEW    
 $RELOAD
  3 250  ; write restart file every 100th timestep
#PROCESS
 $PCS_TYPE
  MASS_TRANSPORT
 $NUM_TYPE
   NEW    
 $RELOAD
  3 250  ; write restart file every 100th timestep
#PROCESS
 $PCS_TYPE
  MASS_TRANSPORT
 $NUM_TYPE
   NEW    
 $RELOAD
  3 250  ; write restart file every 100th timestep
#PROCESS
 $PCS_TYPE
  MASS_TRANSPORT
 $NUM_TYPE
   NEW    
 $RELOAD
  3 250  ; write restart file every 100th timestep
#PROCESS
 $PCS_TYPE
  MASS_TRANSPORT
 $NUM_TYPE
   NEW    
 $RELOAD
  3 250  ; write restart file every 100th timestep
#PROCESS
 $PCS_TYPE
  MASS_TRANSPORT
 $NUM_TYPE
   NEW    
 $RELOAD
  3 250  ; write restart file every 100th timestep
#PROCESS
 $PCS_TYPE
  MASS_TRANSPORT
 $NUM_TYPE
   NEW    
 $RELOAD
  3 250  ; write restart file every 100th timestep
#STOP
