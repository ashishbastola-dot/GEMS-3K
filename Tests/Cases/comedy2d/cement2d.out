GeoSys-OUT: Output ------------------------------------------------
#OUTPUT
 $NOD_VALUES
  HEAD
  1-C 
  2-Ca 
  3-Cl 
  4-H 
  5-Na 
  6-O 
  7-Oxa 
  8-Si 
  9-Zz 
 $ELE_VALUES
  POROSITY
  VELOCITY1_X
  VELOCITY1_Y
 $GEO_TYPE
  DOMAIN
 $DAT_TYPE
  VTK
 $TIM_TYPE
   STEPS 30
#STOP

#OUTPUT // profile
 $PCS_TYPE
  GROUNDWATER_FLOW
 $ELE_VALUES
  VELOCITY1_X
  VELOCITY1_Y
  VELOCITY1_Z
  VELOCITY
 $GEO_TYPE
  POLYLINE LINE_1  
 $DAT_TYPE
  TECPLOT
 $TIM_TYPE
  STEPS 1

#STOP

   100.0
   1000.0
   10000.0
   21000.0
   140700.0
   
   
#OUTPUT // profile
 $NOD_VALUES
  HEAD
  Cl-
 $GEO_TYPE
  POLYLINE LINE_1
 $DAT_TYPE
  TECPLOT
 $TIM_TYPE
  STEPS 40
  

#STOP
